pytorch_lightning/tuner/auto_gpu_select.py

```diff
@@ -22,15 +22,18 @@ def pick_multiple_gpus(nb: int) -> List[int]:
     """
     Raises:
         MisconfigurationException:
-            If ``gpus`` is set to 0, when ``auto_select_gpus=True``.
+            If ``gpus`` or ``devices`` is set to 0, when ``auto_select_gpus=True``.
     """
     if nb == 0:
         raise MisconfigurationException(
-            "auto_select_gpus=True, gpus=0 is not a valid configuration."
+            "auto_select_gpus=True, gpus/devices=0 is not a valid configuration."
```

Comment:
isn't this going to fail in your other PR? where you are checking devices = [], 0, '0'?
not sure which one comes first.

Reply:
One GPU test is failing for this PR, we would need the other one to be merged first!

Reply:
@rohitgr7